site/_includes/layouts/base.njk

```diff
@@ -65,6 +65,8 @@
       })();
     </script>
     {% endif %}
+    
+    {% block chromescripts %}{% endblock %}
```

Comment:
I'd prefer not to move the `chromescripts` to the `<head>` since they are not critical and, even though they are defered, this can still cause them to be fetched earlier than necessary.

```suggestion
```